### ISSUE TITLE
Error when loading the model on HuggingFace

### ISSUE THREAD

**Issue by nairaxo** [speechbrain/speechbrain#1341]
Hello!

I pushed my models on HuggingFace and everything worked well until today. I'm getting these error and I saw that there are the same thing for the SpeechBrain models on HuggingFace. Is it related to "facebook/..." models, so I have nothing to do in my side, do I? 

![issue-speechbrain](https://user-images.githubusercontent.com/81855567/160191882-1176dd0f-9192-4cc4-bca7-0b14efe9a867.jpeg)

**Comment by TParcollet** [comment]
Hum, indeed HF problem I believe.

**Comment by nairaxo** [comment]
Yeah, it definitely comes from HF. So I'm waiting until they fix it!
